src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -46,20 +46,22 @@ class SearchControls extends Component {
     }
 
     return (
-      <div className="search-controls-container">
-        <h4>Find a VA Facility</h4>
-        <div>Search for facilities near you or for a specific service or benefit.</div>
-        <form className="usa-form">
-          <label htmlFor="Street, City, State or Zip">Enter Street, City, State or Zip</label>
-          <input ref="searchField" name="streetCityStateZip" type="text" onChange={this.handleQueryChange} value={currentQuery.searchString}/>
-          <label htmlFor="serviceType">Service Type</label>
-          <select name="services" defaultValue="all" onChange={this.handleFacilityTypeChange}>
-            <option value="all">All</option>
-            <option value="health">Health</option>
-            <option value="benefits">Benefits</option>
-            <option value="cemeteries">Cemeteries</option>
-          </select>
-          <input type="submit" className="full-width" value="Search" onClick={this.handleSearch}/>
+      <div className="search-controls-container clearfix">
+        <form>
+          <div className="columns medium-5">
+            <label htmlFor="Street, City, State or Zip">Enter Street, City, State or Zip</label>
+            <input ref="searchField" name="streetCityStateZip" type="text" onChange={this.handleQueryChange} value={currentQuery.searchString}/>
+          </div>
+          <div className="columns medium-5">
+            <label htmlFor="serviceType">Service Type</label>
+            <select name="services" onChange={this.handleFacilityTypeChange} value={currentQuery.serviceType}>
+              <option value="all">All</option>
+              <option value="va_health_facility">Health</option>
+              <option value="va_benefits_facility">Benefits</option>
+              <option value="va_cemetery">Cemeteries</option>
+            </select>
```

Comment:
There should be a "service type" `select` here that has "primary care", "mental health" and "more services coming soon"